src/hypermedia-constants.js

```diff
@@ -189,7 +189,8 @@ export const Rels = {
 			selectedCourseMergeOfferings: 'https://ipsis.api.brightspace.com/rels/selected-course-merge-offerings',
 			selectedCourseMergeOfferingsCount: 'https://ipsis.api.brightspace.com/rels/selected-course-merge-offerings-count',
 			mergedCourseOfferings: 'https://ipsis.api.brightspace.com/rels/merged-course-offerings',
-			courseMergeLogs: 'https://ipsis.api.brightspace.com/rels/course-merge-logs'
+			courseMergeLogs: 'https://ipsis.api.brightspace.com/rels/course-merge-logs',
+			timeFilters: 'https://ipsis.api.brightspace.com/rels/time-filters',
```

Comment:
Can we split out the time filter stuff now that we've split the stories

Reply:
Removed time-filter